### ISSUE TITLE
Scenario with dervied dataset shouldn't save a scaler

### ISSUE THREAD

**Issue by grdw** [quintel/etmodel#2321]
There's a bit of a crease in the current implementation of the scaling in relation to local datasets. When I create a scenario with a derived dataset, there's an extra scaler object stored with the scenario. This should not be the case since that would imply that the scenario is scaled through the normal interface.

Now I'm trying to find out where the scenario is saved (so when `Api::Scenario.save` ? is called) but I can't seem to find where that happens (for a non-saved scenario). I know that there's a bit of code which looks like that in the `load` method of the `ScenariosController` but I think the only point in the flow of ETmodel when you reach that code is when you're trying to load a saved scenario and that's not the case when you're not saving your scenario.

---

TLDR; Scaler should not be stored while creating a scenario with a derived dataset, where in the code is a scenario saved to ETEngine from ETModel?

**Comment by antw** [comment]
> TLDR; Scaler should not be stored while creating a scenario with a derived dataset, where in the code is a scenario saved to ETEngine from ETModel?

In [javascripts/lib/app.coffee](https://github.com/quintel/etmodel/blob/2c9b4514e42a793611e38830dc51d9f4a0fbe641/app/assets/javascripts/lib/app.coffee#L17-L33), via [ETPlugin](https://github.com/quintel/etplugin/blob/c668ad44d6db6eba2f9b8ec0c0b0f7a1a0ffaa6c/vendor/assets/javascripts/jquery.etmodel.coffee#L261-L293) (an old JS library for communicating with ETE).

**Comment by grdw** [comment]
Closed with https://github.com/quintel/etengine/pull/900
